src/aggregator/aggregator/aggregator.go

```diff
@@ -133,24 +135,39 @@ func NewAggregator(opts Options) Aggregator {
 	iOpts := opts.InstrumentOptions()
 	scope := iOpts.MetricsScope()
 	timerOpts := iOpts.TimerOptions()
+	logger := iOpts.Logger()
+
+	timedForResendEnabledRollupRegexes := make([]*regexp.Regexp, len(opts.TimedForResendEnabledRollupRegexes()))
```

Comment:
nit: `timedForResendEnabledRollupRegexps`

Reply:
hmm I think you need `make([], 0, len(..)` otherwise you're just appending all the regexs after a bunch of nils.

Reply:
it's unfortunate we don't catch this in the unit tests.

Reply:
Good catch. Maybe I can change the unit test to use this NewAgg func to capture this